dev/run-tests.py

```diff
@@ -583,6 +589,7 @@ def main():
     modules_with_python_tests = [m for m in test_modules if m.python_test_goals]
     if modules_with_python_tests:
         run_python_tests(modules_with_python_tests, opts.parallelism)
+        run_python_packaging_tests()
```

Comment:
It's really hard to determine if the Python change is one thats going to break packaging - and the packaging tests are really quite fast. I think for now erring on the side of testing slightly more often than we need is the best course of action.